### PR TITLE
[DOCFIX] Update the UFS extension documentation

### PR DESCRIPTION
**Pull request by LuQQiu**
Update the UFS extension documentations

### CODE REVIEW

File: docs/en/ufs/Ufs-Extension-API.md [hunk 1]
```diff
@@ -45,12 +45,13 @@ to enable a storage system to serve as Alluxio's underlying storage.
 Step 1: Implement the interface `UnderFileSystem`
 
 The `UnderFileSystem` interface is defined in the module `org.alluxio:alluxio-core-common`. Choose
-to extend either `BaseUnderFileSystem` or `ObjectUnderFileSystem` to implement the `UnderFileSystem`
-interface. `ObjectUnderFileSystem` is suitable for connecting to object storage and abstracts away
+to extend either `ConsistentUnderFileSystem` or `ObjectUnderFileSystem` to implement the `UnderFileSystem`
+interface. `ConsistentUnderFileSystem` is used for storage like HDFS which does not have eventual consistency issue.
```
Comment (madanadit) [inline docs/en/ufs/Ufs-Extension-API.md]: nit: make these 2 (consistent and object) bullets. 
which does not have eventual consistency issue -> which is not eventually consistent

File: docs/en/ufs/Ufs-Extension-API.md [hunk 2]
```diff
@@ -130,18 +131,32 @@ In addition, to avoid collisions, specify scope for the dependency `alluxio-core
...
+
+Install the tarball to Alluxio:
+
+```bash
+/bin/alluxio extensions install <path>/<to>/<probject>/target/alluxio-underfs-<ufsName>-<version>.jar
 ```
 
+### Test the Under Storage Extension
+
+To ensure the new under storage develop correctly and fulfill the minimum requirements to work with Alluxio,
```
Comment (madanadit) [inline docs/en/ufs/Ufs-Extension-API.md]: nit: 
- under storage -> under storage module
- UFS tests -> contract tests to test different workflows with various combinations of operations against the under storage

File: docs/en/ufs/Ufs-Extension-API.md [hunk 3]
```diff
@@ -130,18 +131,32 @@ In addition, to avoid collisions, specify scope for the dependency `alluxio-core
...
+/bin/alluxio extensions install <path>/<to>/<probject>/target/alluxio-underfs-<ufsName>-<version>.jar
 ```
 
+### Test the Under Storage Extension
+
+To ensure the new under storage develop correctly and fulfill the minimum requirements to work with Alluxio, 
+one can run UFS tests to issue different UFS workflow combinations to the created under storage.
+
+```bash
+/bin/alluxio runUfsTests --path <scheme>://<path>/ -D<key>=<value>
```
Comment (madanadit) [inline docs/en/ufs/Ufs-Extension-API.md]: nit: add `.` before bin

File: docs/en/ufs/Ufs-Extension-API.md (hunk 1 above)
Comment (LuQQiu) [reply docs/en/ufs/Ufs-Extension-API.md]: Done

File: docs/en/ufs/Ufs-Extension-API.md (hunk 2 above)
Comment (LuQQiu) [reply docs/en/ufs/Ufs-Extension-API.md]: Changed, thanks

File: docs/en/ufs/Ufs-Extension-API.md (hunk 3 above)
Comment (LuQQiu) [reply docs/en/ufs/Ufs-Extension-API.md]: Done

**Comment by LuQQiu** [comment]
@madanadit Addressed the comments, PTAL, thanks

**Comment by LuQQiu** [comment]
jenkins, test this please

File: docs/en/ufs/Ufs-Extension-API.md [hunk 4]
```diff
@@ -130,18 +131,32 @@ In addition, to avoid collisions, specify scope for the dependency `alluxio-core
...
+
+Install the tarball to Alluxio:
+
+```bash
+./bin/alluxio extensions install <path>/<to>/<probject>/target/alluxio-underfs-<ufsName>-<version>.jar
 ```
 
+### Test the Under Storage Extension
+
+To ensure the new under storage module develop correctly and fulfill the minimum requirements to work with Alluxio,
```
Comment (madanadit) [inline docs/en/ufs/Ufs-Extension-API.md]: nit:  sorry i missed this before, change this to "To ensure the new under storage module fulfills the ..."

File: docs/en/ufs/Ufs-Extension-API.md [hunk 5]
```diff
@@ -45,12 +45,13 @@ to enable a storage system to serve as Alluxio's underlying storage.
 Step 1: Implement the interface `UnderFileSystem`
 
 The `UnderFileSystem` interface is defined in the module `org.alluxio:alluxio-core-common`. Choose
-to extend either `BaseUnderFileSystem` or `ObjectUnderFileSystem` to implement the `UnderFileSystem`
-interface. `ObjectUnderFileSystem` is suitable for connecting to object storage and abstracts away
+to extend either **`ConsistentUnderFileSystem`** or **`ObjectUnderFileSystem`** to implement the `UnderFileSystem`
+interface. `ConsistentUnderFileSystem` is used for storage like HDFS which is not eventually consistent.
```
Comment (madanadit) [inline docs/en/ufs/Ufs-Extension-API.md]: i meant an enumeration using `-` starting after "interface."

Comment (LuQQiu) [reply docs/en/ufs/Ufs-Extension-API.md]: Changed to use enumeration, thanks

File: docs/en/ufs/Ufs-Extension-API.md (hunk 4 above)
Comment (LuQQiu) [reply docs/en/ufs/Ufs-Extension-API.md]: Done, thanks

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Comment by madanadit** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**